src/recipe.rs

```diff
@@ -80,7 +81,8 @@ fn build_dependencies(args: &CookArgs) {
         package,
         workspace,
         offline,
-        ..
```

Comment:
I removed this so the compiler would help direct new contributors to where they should add command functionality if they add a new field, but I'm happy to revert this change if it doesn't match the project idioms.